src/sentry/api/endpoints/accept_organization_invite.py

```diff
@@ -83,7 +83,7 @@ def get(
             # When SSO is required do *not* set a next_url to return to accept
             # invite. The invite will be accepted after SSO is completed.
             url = (
-                reverse("sentry-accept-invite", args=[member_id, token])
+                reverse("sentry-accept-invite-with-org", args=[organization_slug, member_id, token])
```

Comment:
It'll be nice to have test coverage for this edge case.

Reply:
it's tested here https://github.com/getsentry/sentry/blob/ba8b94234630ab7ec3aaa7e9617cc6d34ad13796/tests/sentry/api/endpoints/test_accept_organization_invite.py#L99

Reply:
@cathyteng17 I meant that when `url` has been updated here. Generally there should be a test to check for that `url` value.